### ISSUE TITLE
Change search help text headings to h2

### ISSUE THREAD

**Issue by erinchambers** [CDRH/orchid#292]
Currently the headings under Search Help (`en.yml`) are `<h4>`, and the only previous heading is the page title `<h1>`, which means they skip heading levels. 

I'd recommend we make them `<h2>`s and making the font size smaller either through a CSS style or a Bootstrap class (not available in 3.4, so we'll have to wait until migrating to a newer Bootstrap version)

**Comment by erinchambers** [comment]
For Petitioning for Freedom I used this code - dropping it here so we can copy/paste.

**In `en.yml`:**

```
<h2 class="h4">Basic Search</h2>
<p>Simply enter the word you wish to find and the search engine will search for every instance of the word in the journals. For example: Fight. All instances of the use of the word fight will show up on the results page.</p>
<h2 class="h4">Wildcard Searches</h2>
<p>Using an asterisk (*) will increase the odds of finding the results you are seeking. For example: Fight*. The search results will display every instance of fight, fights, fighting, etc. More than one wildcard may be used. For example: *ricar*. This search will return most references to the Aricara tribe, including Ricara, Ricares, Aricaris, Ricaries, Ricaree, Ricareis, and Ricarra. Using a question mark (?) instead of an asterisk (*) will allow you to search for a single character. For example, r?n will find all instances of ran and run, but will not find rain or ruin.</p>
<h2 class="h4">Capitalization</h2>
<p>Searches are not case sensitive. For example: george will come up with the same results as George.</p>
<h2 class="h4">Phrase Search</h2>
<p>Searching for a specific phrase may help narrow down the results. Rather long phrases are no problem. For example: "This white pudding we all esteem".</p>
<h2 class="h4">Subsequent Searches</h2>
<p>Because of the creative spellings used by the journalists, it may be necessary to try your search multiple times. For example: P?ro*. This search brings up numerous variant spellings of the French word pirogue, "a large dugout canoe or open boat." Searching for P?*r*og?* will bring up other variant spellings. Searching for canoe or boat also may be helpful.</p>
```

**In custom scss (`$font-size-h4` variable is set in `bootstrap-variables.scss`):**

```
.h4 {
  font-size: $font-size-h4;
}
```

**Comment by erinchambers** [comment]
IMPORTANT! These should be `<h3>`s, not `<h2>`; the button to expand the search help box is an `<h2>`. Need to fix this on PFF as well as the rest.
